README.md

```diff
@@ -67,38 +104,33 @@ reMarkable screen sharing over SSH.
 
 If you have problems, don't hesitate to [open an issue](https://github.com/rien/reStream/issues/new) or [send me an email](mailto:rien.maertens@posteo.be).
 
-## Requirements
+## Extra Dependencies
 
 On your **host** machine:
 
-- Any POSIX-shell (e.g. bash)
-- ffmpeg (with ffplay)
-- ssh
-- Video4Linux loopback kernel module if you want to use `--webcam`
+- Video4Linux Loopback kernel module if you want to use `--webcam`
 - netcat if you want to use `--unsecure-connection`
 
-On your **reMarkable** you need the `restream` binary (see [installation instructions](#installation)).
-
 ### Video4Linux Loopback
 
 To set your remarkable as a webcam we need to be able to fake one. This is where the Video4Linux Loopback kernel module comes into play. We need both the dkms and util packages. On Ubuntu you need to install:
 
 ```
-apt install v4l2loopback-utils v4l2loopback-dkms
+# apt install v4l2loopback-utils v4l2loopback-dkms
 ```
 
 In some package managers `v4l2loopback-utils` is found in `v4l-utils`.
 
 After installing the module you must enable it with
 
 ```
-modprobe v4l2loopback
+# modprobe v4l2loopback
 ```
 
 To verify that this worked, execute:
 
 ```
-v4l2-ctl --list-devices
+$ v4l2-ctl --list-devices
```

Comment:
Can `v4l2-ctl` be executed as user? Because it starts with an `$`?

Reply:
yes